build.gradle.kts

```diff
@@ -79,12 +69,10 @@ dependencies {
     // Stats
     implementation("org.bstats:bstats-bukkit:3.0.2")
     // Commands
-    implementation("cloud.commandframework:cloud-annotations:1.8.3")
-    implementation("cloud.commandframework:cloud-minecraft-extras:1.8.3")
-    implementation("cloud.commandframework:cloud-paper:1.8.3")
-    annotationProcessor("cloud.commandframework:cloud-annotations:1.8.3")
-    implementation("me.lucko:commodore:2.2")
-
+    implementation("org.incendo:cloud-annotations:2.0.0-rc.2")
+    implementation("org.incendo:cloud-minecraft-extras:2.0.0-beta.8")
+    implementation("org.incendo:cloud-paper:2.0.0-beta.8")
+    annotationProcessor("org.incendo:cloud-annotations:2.0.0-rc.2")
```

Comment:
Why is beta and RC mixed?

Reply:
cloud-paper and cloud-minecraft-extras do not have release candidates yet
those all are different projects actually